building-with-ocean/obtaining-api-key.md

```diff
@@ -0,0 +1,7 @@
+# Obtaining API key
```

Comment:
This doesn't need to be in a separate file, you may as well move it to the section in `configuration.md`. I'd also ideally want more explanation - a brief couple of sentences on what this API key is for and why it is needed. At the moment it is just a list of links which doesn't mean much to a newcomer.

Reply:
The purpose of keeping this in a separate file is because provider, Aquarius deployment files also point to the content here.

- [x] Added more explanation on why is this needed.